plugins/org.eclipse.thym.android.core/src/org/eclipse/thym/android/core/adt/AndroidSDKManager.java

```diff
@@ -436,12 +436,12 @@ public void startAVDManager() throws CoreException{
 		ExternalProcessUtility processUtility = new ExternalProcessUtility();
 		StringBuilder command = new StringBuilder(getAndroidCommand());
 		command.append(" avd ");
-		processUtility.execAsync(command.toString(), null, null, null, null);
+		processUtility.execAsync(command.toString(), new File(toolsDir), null, null, null);
 	}
 	
 	private String getAndroidCommand(){
-		String command = addQuotes(toolsDir+"android");
-		return isWindows() ? "cmd /c "+ command : command;
+		String command = "android";
```

Comment:
This will cause it to fail on MacOS and linux. On Mac and linux it should be "./android" if we are specifying the working directory. Can we keep the earlier full path? Does that work on Win?